plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPreferencesPanel.java

```diff
@@ -265,15 +268,14 @@ public void handle(Exception ex) {
         MessageDialog.openError(getShell(), Messages.getString("cannot.open.browser"), ex.getLocalizedMessage());
       }
     }));
-    GridLayoutFactory.fillDefaults().generateLayout(promoteComposite);
-  }
-
-  private void createStopPreviousVersionSection() {
-    Composite composite = new Composite(this, SWT.NONE);
 
-    stopPreviousVersionButton = new Button(composite, SWT.CHECK);
+    stopPreviousVersionButton = new Button(promoteComposite, SWT.CHECK);
     stopPreviousVersionButton.setText(Messages.getString("stop.previous.version"));
-    GridLayoutFactory.fillDefaults().generateLayout(composite);
+    GridData stopPreviousVersionLayoutData = GridDataFactory.swtDefaults().create();
+    stopPreviousVersionLayoutData.horizontalIndent = INDENT_CHECKBOX_ENABLED_WIDGET;
+    stopPreviousVersionButton.setLayoutData(stopPreviousVersionLayoutData);
```

Comment:
This can be rewritten in one line as:
```
GridDataFactory.swtDefaults().indent(INDENT_CHECKBOX_ENABLED_WIDGET, 0).applyTo(stopPreviousVersionButton);
```